mat64/dense_test.go

```diff
@@ -227,7 +227,9 @@ func (s *S) TestSetRowColumn(c *check.C) {
 			t.Clone(a)
 			a.SetRow(ri, make([]float64, a.mat.Cols))
 			t.Sub(t, a)
-			c.Check(t.Norm(0), check.Equals, floats.Norm(row, 2))
+			if math.Abs(Norm(t, 2)-floats.Norm(row, 2)) > 1e-14 {
+				c.Errorf("Row norm mismatch")
```

Comment:
Here and below output the row/col given to the floats.Norm and the norms returned by each function.